streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -450,6 +452,24 @@ public void shouldThrowForUnknownTopicDeprecated() {
         }
     }
 
+    @Test
+    public void shouldGetSinkTopicNames() {
+        testDriver = new TopologyTestDriver(setupSourceSinkTopology(), config);
+
+        pipeRecord(SOURCE_TOPIC_1, testRecord1);
+
+        assertThat(testDriver.producedTopicNames(), hasItem(SINK_TOPIC_1));
+    }
+
+    @Test
+    public void shouldGetInternalTopicNames() {
+        testDriver = new TopologyTestDriver(setupTopologyWithTwoSubtopologies(), config);
```

Comment:
This does not setup a topology with an internal topic names. You would need to have a store or use the `StreamsBuilder` with some repartition topic. Or do you not care as you are just interested in the "chaining" itself? (than maybe change the test name?)

Or even better, add tests for all cases, including repartition and changelog? Guess, it could even be one single test that covers all cases at once?

Reply:
done.